src/agent/README.md

```diff
@@ -39,11 +39,21 @@ After that, we drafted the initial code here, and any contributions are welcome.
 ## Getting Started
 
 ### Build from Source
-The rust-agent need to be built with rust newer than 1.37, and static linked with `musl`.
+The rust-agent need to be built with static linked with musl:
 ```bash
-rustup target add x86_64-unknown-linux-musl
-sudo ln -s /usr/bin/g++ /bin/musl-g++  
-cargo build --target x86_64-unknown-linux-musl --release
+$ arch=$(uname -m)
+$ [ "$arch" == "ppc64le" ] && arch=powerpc64le
+$ rustup target add "${arch}-unknown-linux-musl"
+$ sudo ln -s /usr/bin/g++ /bin/musl-g++
+```
+Download the kata-containers source and build the agent:
+```bash
+$ GOPATH="${GOPATH:-$HOME/go}"
+$ dir="$GOPATH/src/github.com/kata-containers"
+$ mkdir -p "$dir" && cd "$dir"
+$ git clone --depth 1 https://github.com/kata-containers/kata-containers && cd kata-containers
+$ cd $GOPATH/src/github.com/kata-containers/kata-containers/src/agent
```

Comment:
please use `pushd` and `popd` after `make`

Reply:
Thanks @devimc 
I will change it to `$ pushd $GOPATH/src/github.com/kata-containers/kata-containers/src/agent && make && popd`

Reply:
actually, pushd and popd is no longer needed, you can use `make -C $GOPATH/src/github.com/kata-containers/kata-containers/src/agent`